openedx/core/djangoapps/coursegraph/tasks.py

```diff
@@ -23,7 +22,7 @@
 bolt_log = logging.getLogger('neo4j.bolt')  # pylint: disable=invalid-name
 bolt_log.setLevel(logging.ERROR)
 
-PRIMITIVE_NEO4J_TYPES = (integer, string, str, float, bool)
+PRIMITIVE_NEO4J_TYPES = (int, bytes, str, float, bool)
```

Comment:
This was a Python 2 compatibility thing. We can just use the raw Python types now.